packages/react-core/src/operations/constants/SpatialIndexTypes.d.ts

```diff
@@ -0,0 +1,6 @@
+export enum SpatialIndex {
+  H3 = 'h3',
+  S2 = 's2',
+  QUADKEY = 'quadkey',
+  QUADINT = 'quadint'
+}
```

Comment:
I think that in TS the enum keys are in pascal case.
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
> I think that in TS the enum keys are in pascal case. https://www.typescriptlang.org/docs/handbook/enums.html

I would say that we are not following that convention in the project
- https://github.com/CartoDB/carto-react/blob/fad57802b35fd7c467e5bfb36b470191f740ebce/packages/react-core/src/filters/FilterTypes.d.ts#L1
- https://github.com/CartoDB/carto-react/blob/2b603f3e86d3d04df4e6232768a4bb62c891674a/packages/react-redux/src/slices/cartoSlice.d.ts#L57
- https://github.com/CartoDB/carto-react/blob/6b62898ebeaa902d6ddfb17a2e3d37b8eeebafca/packages/react-basemaps/src/basemaps/basemaps.d.ts#L1

Reply:
Yes... I know, but I thought we would do it correctly hereinafter hahaha. But it's ok.